Makefile

```diff
@@ -1,52 +1,22 @@
-all: ci rhel
+ci_keys = keys/verifier-public-key-openshift-ci keys/verifier-public-key-openshift-ci-2
+ci_stores = stores/store-openshift-ci-release
+ci_manifests = manifests/0000_90_cluster-update-keys_configmap.yaml
+rhel_keys = keys/verifier-public-key-redhat-release keys/verifier-public-key-redhat-beta-2
+rhel_stores = stores/store-openshift-official-release stores/store-openshift-official-release-mirror
+rhel_manifests = manifests.rhel/0000_90_cluster-update-keys_configmap.yaml
+
+all: hack/generate-configmap.sh ci rhel
```

Comment:
There is no rule to generate `hack/generate-configmap.sh`, and the `ci` and `rhel` targets depend on it (even if they don't explicitly list it as a dependency yet, as of d8197c0d0a4).  So `all` doesn't need to depend on it here, you can just have `all: ci rhel` for "when you see `all`, that means 'make sure both types of manifests'".

Reply:
Ah, yes. Thank you for the clarification.
Fixed in https://github.com/openshift/cluster-update-keys/pull/39/commits/35be0a0f942954370de1b6350612778ae6174203.